### ISSUE TITLE
Improve preset selection when presets are filtered by URL parameter

### ISSUE THREAD

**Issue by willemarcel** [openstreetmap/iD#7767]
Some users that are a bit more experienced on iD are getting confused with the preset selection.

We should let them know when the presets are limited. Maybe we could also restrict the point, line, area selector according to the preset geometry types.

**Comment by rachblevine** [comment]
Agreed! Thanks for starting this ticket.  Another option is instead of displaying the number of options that are missing when you type in what you are looking for, a note is displayed "you are searching for a feature on a restricted project."

**Comment by quincylvania** [comment]
> We should let them know when the presets are limited. Maybe we could also restrict the point, line, area selector according to the preset geometry types.

Sounds fine. I agree this could be made clearer.

**Comment by willemarcel** [comment]
Another issue is the number of presets when the user searches:

![image](https://user-images.githubusercontent.com/666291/97694981-15dfef00-1a82-11eb-926d-56af50873995.png)

**Comment by mbrzakovic** [comment]
Hi all.
Just to clarify, this is specific to Hot taking manager using internal iD?
If that is the case, could you please provide more context on this issue, and the current state of hotosm-iD mechanics?
From a quick look, i noticed that [hotosm](https://github.com/hotosm/tasking-manager/blob/develop/frontend/package.json) is using [hotosm/iD (fork)](https://github.com/hotosm/iD) so should this fix go to the fork?

I am removing the milestone in order to speed up next release however I would like to have this resolved in the following cycle.
@willemarcel, @rachblevine visibility.

**Comment by willemarcel** [comment]
@mbrzakovic It's not specific to HOT TM. If you load any iD or RapiD with `presets=building/house,building/residential,building` in the URL, the preset restriction will be applied.

The HOT fork was made sometime ago, but now TM uses iD without any modification.

**Comment by mbrzakovic** [comment]
Got it @willemarcel thanks for explanation!
